### DISCUSSION TITLE
[Feature] Recent uploads overview

### DISCUSSION THREAD

**Discussion by coltenkrauter** [immich-app/immich#10701] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I want a recent summary overview with options to filter by user, device, and timeframe. 

This would be nice since, from time to time, I like to be able to validate recent uploads before I delete them from my device. For example, if I have 525 images to delete from my iPhone, it would be nice to pop in and view the recent uploads... filter it by the time frame when my device began uploading the 525 images and ensure that the numbers line up... or are at least close. It is interesting information and it can also provide a sanity check... 

Also, it would be interesting to see this small report on the other users that I have access to their photos... "Sara upload report" and it shows 5 videos and 12 images uploaded this last week.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#8716

**Comment by bdsoha** [comment]
#10398
